docs/robots_file.md

```diff
@@ -0,0 +1,376 @@
+<!--
+# @markup markdown
+# @title Robots file
+-->
+
+# Bed verifications background
+The robots configuration file defines all the bed verifications in Limber.
+
+A bed verification is a check done by the lab staff when loading a liquid handler robot. The idea is to check both that the plates are on the correct beds (plate locations) on the deck of the robot, and if pairing them that those plates have the expected parent and child relationship in the LIMS (that you've not muddled up pairings of plates).
+The goal is to reduce human error when loading liquid handler robots, preventing mistakes and plate swaps. This is critical in high throughput pipelines especially, where many plates are being processed simultaneously in repetitive tasks.
+
+There will be a location barcode stuck on or near each bed on the liquid handler deck. The user scans both this bed barcode and the labware barcode, and the system validates that the labware is of the correct purpose and is in the correct deck location. Clicking on 'validate' then checks the labwares are correctly related to one another.
+
+A successful bed verification typically happens in 2 stages; validation, then transfers of samples to the child.
+
+The use of the bed verification is also logged as an event to the MLWH.
+
+Remember to include any bed verifications in your Integration Suite tests. They are an important part of pipeline development and vital to prevent mistakes in the labs.
```

Comment:
This is the first mention of the Integration Suite, is it worth including its usage somewhere else

Reply:
Probably a good idea to add a Pipeline testing section somewhere, so yes good idea.

Reply:
added